.github/workflows/testSqlDelight.yml

```diff
@@ -0,0 +1,44 @@
+name: SQLDelight Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/checkout@v3
+      with:
+        repository: 'cashapp/sqldelight'
+        path: 'sqldelight'
+
+    - name: Validate Gradle Wrapper
+      uses: gradle/wrapper-validation-action@v1
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3.4.1
+      with:
+        distribution: 'zulu'
+        java-version: 11
+
+    - name: Setup gradle
+      uses: gradle/gradle-build-action@v2
+
+    - name: Publish locally
+      run: ./gradlew publishAllPublicationsToInstallLocallyRepository
+
+    - name: Copy sql-psi to sqldelight local repo
+      run: mkdir -p sqldelight/build/localMaven && mv build/localMaven sqldelight/build/localMaven
+
+    - name: Get the version of sql-psi
+      run: ./gradlew printVersion && export sqlPsiVersion=$(cat version) && rm version
+    - name: Build SqlDelight
+      run: ./gradlew -PsqlPsiVersion=$sqlPsiVersion build
```

Comment:
On second thought, I'm not sure this is a great idea because the SQLDelight testing workflow is INSANE compared to sql-psi. It runs docker, an android emulator, runs on linux, windows, and mac, builds a sample app, etc. I'd rather not introduce that complexity here where the build is a simple jvm build.

Reply:
I already expected this kind of comment :D So what do we want to test? The dialect and compiler only? No integration tests, nothing else: `dialects:build sqldelight-compiler:build`